src/test/java/org/jvnet/hudson/test/RealJenkinsRuleSyntheticPluginTest.java

```diff
@@ -40,7 +41,7 @@ public final class RealJenkinsRuleSyntheticPluginTest {
     @Rule public RealJenkinsRule rr = new RealJenkinsRule().prepareHomeLazily(true);
 
     @Test public void smokes() throws Throwable {
-        rr.addSyntheticPlugin(Stuff.class.getPackage()).done();
+        rr.addSyntheticPlugin(new SyntheticPlugin(Stuff.class.getPackage()));
```

Comment:
The change is no longer backwards compatible, but this is what needs to be done to adapt.